sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -268,6 +268,15 @@ private[spark] object QueryCompilationErrors {
       s"(valid range is [1, $size])", t.origin.line, t.origin.startPosition)
   }
 
+  def groupByPositionRefersToAggregateFunctionError(
+      index: Int,
+      expr: Expression,
+      t: TreeNode[_]): Throwable = {
+    new AnalysisException(s"GROUP BY expression in position $index is an aggregate function. " +
```

Comment:
`is an aggregate function` => `is or contains an aggregate function`

Reply:
> `is an aggregate function` => `is or contains an aggregate function`

Done